am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/FilterResourceWithSecurityClassification.java

```diff
@@ -0,0 +1,364 @@
+package integration.uk.gov.hmcts.reform.amlib;
```

Comment:
rename this file FilterResourceWithSecurityClassificationTest.java, it is probably not being picked up as a test class at the moment

Reply:
This is being pick up .I have seen it in local test report.

Reply:
okay but as good practice all test classes should end with Test

Reply:
agreed, lets rename to be consistent and follow the existing  naming conventions

Reply:
done